From 06377061aa7bc13e72cb8516ab0743ee7d3de9b3 Mon Sep 17 00:00:00 2001
From: Dominic Hargreaves <dom@earth.li>
Date: Sun, 27 Nov 2011 16:48:07 +0000
Subject: Disable failing GNU/Hurd tests dist/IO/t/io_pipe.t

These tests currently fail on GNU/Hurd, so disable them

Bug-Debian: http://bugs.debian.org/650096

Patch-Name: debian/hurd_test_skip_io_pipe.diff
---
 dist/IO/t/io_pipe.t |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/dist/IO/t/io_pipe.t b/dist/IO/t/io_pipe.t
index b7897bb..6e4768d 100644
--- a/dist/IO/t/io_pipe.t
+++ b/dist/IO/t/io_pipe.t
@@ -24,6 +24,9 @@ BEGIN {
     elsif ($^O eq 'MSWin32' && !$ENV{TEST_IO_PIPE}) {
 	$reason = 'Win32 testing environment not set';
     }
+    elsif ($^O eq 'gnu') {
+        $reason = 'fails on GNU/Hurd (Debian #650096)';
+    }
     if ($reason) {
 	print "1..0 # Skip: $reason\n";
 	exit 0;
